src/main/java/Luke.java

```diff
@@ -0,0 +1,205 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Luke {
+    public static void main(String[] args) throws LukeException {
+        // Greetings
+        System.out.println("________________________________________________________________________");
+        System.out.println("Hello! I'm Luke");
+        System.out.println("What can I do for you?");
+        System.out.println("________________________________________________________________________");
+
+        // User inputs
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        ArrayList<Task> list = new ArrayList<>();
+
+        // Conditions
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
+                System.out.println("________________________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+
+                for (int i = 0; i < list.size(); i++) {
+                    if (list.get(i) != null) {
+                        System.out.println(i + 1 + "." + list.get(i));
+
+                    }
+                }
+                System.out.println("________________________________________________________________________");
+
+            } else if (input.contains("mark")) {
```

Comment:
Might be a redundant if check. You can just split the string and verify if the mark/ unmark is present, while also ensuring the length of the split instruction is 2. This way, you confirm the task name's presence without extra checks. This might be applicable for other commands as well.